server/build/release.mk

```diff
@@ -156,16 +156,13 @@ package-general:
 	mkdir -p $(DIST_PATH_GENERIC)/logs
 	mkdir -p $(DIST_PATH_GENERIC)/prepackaged_plugins
 
-	@# Copy binary
+	@# Copy binaries
 ifeq ($(BUILDER_GOOS_GOARCH),"$(CURRENT_PACKAGE_ARCH)")
-	cp $(GOBIN)/$(MM_BIN_NAME) $(DIST_PATH_GENERIC)/bin # from native bin dir, not cross-compiled
+	cp $(GOBIN)/$(MM_BIN_NAME) $(GOBIN)/$(MMCTL_BIN_NAME) $(DIST_PATH_GENERIC)/bin # from native bin dir, not cross-compiled
```

Comment:
`build-cmd-*` targets are already building `./cmd/...`, so `mmctl` will be included in the `$(GOBIN)` without needing to update anything. We just need to remove the `scripts/download_mmctl_release.sh` call and make sure that the binary is copied over with the `mattermost` one for the build to include it